compiler/luci/lang/include/luci/IR/Nodes/CircleDanglingNode.h

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright (c) 2021 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef __LUCI_IR_CIRCLE_DANGLING_NODE_H__
+#define __LUCI_IR_CIRCLE_DANGLING_NODE_H__
+
+namespace luci
+{
+
+/**
+ * @brief Virtual CircleDanglingNode in Circle used as dangling input to CircleNode
+ */
+class CircleDanglingNode final
+  : public FixedArityNode<0, CircleNodeImpl<CircleOpcode::CIRCLEDANGLINGNODE>>
+{
+public:
+  CircleDanglingNode() = default;
+};
```

Comment:
This new virtual node helps with the import of SVDF op. 
SVDF has one specific input tensor, that doesn't store any data (its buffer is empty), but is used in the kernel (as a scratch tensor) and stores quantization parameters, type and shape.
That is why we use this new dangling node to store tensor attribute, and use it as input node to SVDF.

Reply:
If you introduce this virtual node, you must change all codes that handle virtual nodes.